docs/fieldnorm/requirements.md

```diff
@@ -0,0 +1,20 @@
+# Requirements in software development
+The complexity and variety of developments in the field of bioinformatics, comes with a need for clear prerequisites before the start of a project. These requirements of software should describe the prerequisites that the software should meet and they should be clear before starting on a project. The requirements may relate to several aspects, including:
+- the performance of the desired software
+- the physical characteristics
+- the computing environment
+- a need for compatibility
+- input- and output-formats
+- interfaces to other systems
+- communication from the software
+- security requirements
+- user interfaces to the software
+- data definitions
+- installation and acceptance requirements
+- requirements related to the methods of operation and maintenance
+- network-related requirements
+- user maintenance requirements
+- regulatory requirements
+
+All software development projects should have a standardized and documented approach for gathering requirements. Requirements can come from end-users, bio-informaticians and other stakeholders. It is recommended to express the requirements in a form that is SMART (specific, measurable, attainable, realistic and time-bound). It is essential to document these requirements clearly and track them throughout the development process (e.g. in JIRA, Confluence or GitHub/GitLab). Once these requirements have been implemented as features in the software, they should be validated before the release is used in production. Any changes or updates to these requirements should be carefully reviewed and documented.
```

Comment:
"It is essential to document these requirements clearly and track..." I would refrain from using words like "clearly" as this could have a lot of different implications to what "clearly" means. Maybe just put it down like "Requirements should be documented in a way that allows them to be tracked throughout..."